src/s2e/Plugins/Lua/LuaAnnotationState.cpp

```diff
@@ -17,6 +17,7 @@ const char LuaAnnotationState::className[] = "LuaAnnotationState";
 
 Lunar<LuaAnnotationState>::RegType LuaAnnotationState::methods[] = {
     LUNAR_DECLARE_METHOD(LuaAnnotationState, setSkip),
+    LUNAR_DECLARE_METHOD(LuaAnnotationState, skipInstruction),
```

Comment:
You would need to have a LuaFunctionAnnotationState/LuaInstructionAnnotationState with appropriate methods in both.